pkgs/development/haskell-modules/configuration-ghc-9.8.x.nix

```
@@ -48,27 +49,119 @@ self: super: {
   unix = null;
   xhtml = null;
 
-  # https://github.com/tibbe/unordered-containers/issues/214
-  unordered-containers = dontCheck super.unordered-containers;
+  aeson = super.aeson_2_2_1_0;
```

Version changes should always use `self`, so any changes to the versioned attribute in the same overlay still get applied. This problem is present basically everywhere in this change, so I'm noting it only here.

It is also good practice to put `doDistribute` in front, so Hydra won't skip creating a job entry for this attribute if it is explicitly requested.

```suggestion
  aeson = doDistribute self.aeson_2_2_1_0;
```